cmd/kube-proxy/app/server.go

```diff
@@ -299,8 +299,19 @@ func (s *ProxyServer) Run() error {
 	// Tune conntrack, if requested
 	if s.Conntracker != nil {
 		if s.Config.ConntrackMax > 0 {
-			if err := s.Conntracker.SetMax(int(s.Config.ConntrackMax)); err != nil {
-				return err
+			err := s.Conntracker.SetMax(int(s.Config.ConntrackMax))
+			if err != nil {
+				if err != readOnlySysFSError {
+					return err
+				}
+				// readOnlySysFSError is caused by a known docker issue (https://github.com/docker/docker/issues/24000),
+				// the only remediation we know is to restart the docker daemon.
+				// Here we'll send an node event with specific reason and message, the
+				// administrator should decide whether and how to handle this issue,
+				// whether to drain the node and restart docker.
+				// TODO(random-liu): Remove this when the docker bug is fixed.
+				s.Recorder.Eventf(s.Config.NodeRef, api.EventTypeWarning, err.Error(),
+					"Kube-proxy can't apply system configuration properly, becasue docker unexpectedly mounts sysfs as read-only for privileged container (docker issue #24000). To remedy this please restart the docker daemon.")
```

Comment:
typo: s/becasue/because/

Can we shorten this?  Maybe "Kube-proxy can't apply system configuration: /sys is read-only (docker issue #24000).  To remedy, restart docker."